canonical-web-and-design/jaas-dashboard
# Update icon models color

Fixes #357

## Done

Update icon models color

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check icon color
